### ISSUE TITLE
Broaden accessibility reference

### ISSUE THREAD

**Issue by brewerj** [w3ctag/ethical-web-principles#10]
The current statement about accessibility is very limited: "We must make our websites accessible for people with disabilities." This should be broadened to better reflect what's needed for an accessible web, as well as the breadth of the accessibility work at W3C. People with disabilities not only need accessibility in websites, we need accessibility in any content and apps, in tools that create content, in tools that render content, in how authentication happens, in our specs, in the tools that we use to develop our specs, etc. 
Proposed rewording: "We must make our websites, apps, authoring tools, browsers, specs, and development tools accessible for people with disabilities."

**Comment by lknik** [comment]
Can we come up with a single (or two) descriptive words to catch it all?
I worry that if we choose to list a lot of different elements we might still omit some and that will make an even stronger (than now) impression of exclusion.

I'm all for specific and comprehensive approach here, like with security/privacy for features.

**Comment by torgo** [comment]
In some disucussion with @hadleybeeman today on this. Because we want to keep this high level our suggestion is to change the wording slightly. Instead of "websites" how about using a more broad term like "We must ensure that everything we write and build for web platform is acccessible for people with disabilities." (with some additional reference links showing where this work is taking place).

**Comment by alice** [comment]
On further thought, I think @torgo's rewording misses some of @brewerj's original point, regarding the inclusion of people with disabilities in all web-related activities rather than just as users.

How about this as a basis for a more general rephrasing?

"The web platform and the tools used to create it must be accessible to people with disabilities. This encompasses the spec creation process, user agent development, authoring for the web, and all parts of the end user experience."

**Comment by hadleybeeman** [comment]
@alice and @brewerj: @torgo  and I are looking at this again.  Trying to simplify the language a little, would this meet your needs? 

"The web platform and the tools we use to create it must be accessible to people with disabilities. They should be able to meaningfully participate in the creation of specs, user agents, content and the end user experience."

**Comment by alice** [comment]
Slight edit for clarity: 

"The web platform and the tools we use to create it must be accessible to people with disabilities. They should be able to meaningfully participate in the creation of specs, user agents and content, and the platform should enable a fully accessible end user experience."

**Comment by hadleybeeman** [comment]
[Pull request](https://github.com/w3ctag/ethical-web-principles/pull/15) with suggested edits. Thanks for everyone's help with this!

**Comment by brewerj** [comment]
Thanks for the additional work on the accessibility language, which I think now works better for a high-level approach on ethical web principles. Two comments:

* while "web platform and the tools we use to create it" is clearly more comprehensive than the original focus on web content alone (thanks), I'm wondering how clear it would be to the intended audience that this also includes content on the web platform. This could be clarified by saying "Web content, the web platform, and the tools we use to create these must be accessible to people with disabilities."

* "They" at the beginning of the second sentence initially reads as though it is referring back to the web platform and tools. On re-reading, it becomes clearer that the "they" is referring to people with disabilities. Where possible I try to avoid using "they" for people with disabilities, since we're so often positioned as "the other" (though I notice that "they" is also used to refer to privacy & security users). Breaking up the two sentences differently may increase readability, while avoidong this use of "they": 

    "Web content, the web platform, and the tools we use to create it must be accessible to people with disabilities, who should be able to meaningfully participate in the creation of specifications, user agents and content. The platform should enable a fully accessible end user experience."

**Comment by hadleybeeman** [comment]
Thanks, @brewerj! We have looked at both your points, and have [amended the pull request](https://github.com/w3ctag/ethical-web-principles/pull/15). 

We did not intend for people with disabilities to be treated like an "other", so thank you for flagging that. That's a very important point. We've altered the wording to emphasise that we are all in this together.

On your first point, @torgo and I think that the web platform does include the content, so while we do want to support your effort to be inclusive, we would like to leave the words as they are. We are trying hard to keep the language concise and understandable. 

Thanks again for your help!

**Comment by hadleybeeman** [comment]
Addressed in [pull request](https://github.com/w3ctag/ethical-web-principles/pull/15) and merged at [TAG Cupertino face-to-face](https://github.com/w3ctag/meetings/blob/gh-pages/2019/12-cupertino/README.md).
